src/components/tasks/constants.js

```diff
@@ -1,3 +1,4 @@
 export const INITIAL_TASKS_LIMIT = 3;
 export const ADD_MORE_TASKS_LIMIT = 5;
-export const SEE_MORE = 'See More';
\ No newline at end of file
+export const SEE_MORE = 'See More';
+export const TABS = ['ASSIGNED', 'COMPLETED', 'AVAILABLE','IN_PROGRESS', 'NEEDS_REVIEW', 'IN_REVIEW', 'VERIFIED',  'MERGED']
```

Comment:
As this is a TypeScript project I think it's better to use TypeScript files instead of JavaScript like we are doing here. 

One benefit of this would be that  `TABS` variable can be used like this
![image](https://user-images.githubusercontent.com/56453897/220975938-4f0ee058-0a24-4e2a-9c30-0ba7c1d3a1c2.png)

![image](https://user-images.githubusercontent.com/56453897/220976022-a5c060a5-7800-4d32-967f-39e93ab9cc0c.png)

This can be then exported and used in `Tabs/Tabs.tsx` to make the type of  `Tabprops.tabs` event tightly bound to what values are being used here.

Reply:
nicely explained @yesyash